packages/nextra/src/layout.ts

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { SSGContext } from './ssg'
+import { useInternals } from './use-internals'
+
+export default function Nextra(props: any): React.ReactElement {
+  const { context, Layout } = useInternals()
+  const { Content, ...restContext } = context
+
+  return React.createElement(
+    Layout,
+    restContext,
+    React.createElement(
+      SSGContext.Provider,
+      {
+        value: props
+      },
+      React.createElement(Content, props)
+    )
+  )
```

Comment:
any reason to not use JSX markup here?

Reply:
We don't have JSX enabled for the `nextra` package so I just went with this :D